include/bluetooth/mesh/sensor.h

```diff
@@ -258,7 +264,7 @@ struct bt_mesh_sensor_setting {
 	 *
 	 *  @return 0 on success, or (negative) error code otherwise.
 	 */
-	int (*set)(
+	int (*set)(struct bt_mesh_sensor_srv *srv,
 		struct bt_mesh_sensor *sensor,
```

Comment:
Alignment. Several places.

Reply:
Well, this has passed the check patch checks. I had tried aligning at the very beginning and the checks started failing. Then I So I'm leaving this as is.